phpBB/styles/prosilver/template/overall_footer.html

```diff
@@ -66,6 +66,24 @@ <h3 class="alert_title">&nbsp;</h3><p class="alert_text"></p>
 <script src="{T_JQUERY_LINK}"></script>
 <!-- IF S_ALLOW_CDN --><script>window.jQuery || document.write('\x3Cscript src="{T_ASSETS_PATH}/javascript/jquery.min.js?assets_version={T_ASSETS_VERSION}">\x3C/script>');</script><!-- ENDIF -->
 <script src="{T_ASSETS_PATH}/javascript/core.js?assets_version={T_ASSETS_VERSION}"></script>
+{% if S_VIEWTOPIC %}
+	<script>
+		(function($){
+			document.write('\x3Cscript src="{T_ASSETS_PATH}/javascript/jquery.lazyload.1.9.6.min.js">\x3C/script>');
```

Comment:
What do you think about this https://developers.google.com/web/updates/2016/08/removing-document-write

Reply:
This should only have an effect when it’s not deferred or as done here loaded after the dom in the footer

Reply:
The path differs from the actual and the version number also seems to be different (file says it's 1.9.7).

Reply:
seems he messed up the release and double released the same thing under two version numbers 🤪

Reply:
Yeah
![immagine](https://user-images.githubusercontent.com/480857/83344630-d1e99200-a309-11ea-89b3-4abaf6e612ef.png)